server/methods/core/cart.js

```diff
@@ -434,6 +434,10 @@ Meteor.methods({
     // we need to get the parent of the option to check if parcel info is stored there
     const immediateAncestors = variant.ancestors.filter((ancestor) => ancestor !== product._id);
     const immediateAncestor = Collections.Products.findOne({ _id: immediateAncestors[0] });
+    // Get default parcel size
+    const shop = Collections.Shops.findOne({
+      _id: Reaction.getShopId()
```

Comment:
I think we need to look more closely at which shop we should be grabbing here. I think these settings are set at the Marketplace level, not at the individual shop level (at least not yet)

Reply:
@zenweasel to fix this, I used `Reaction.getPrimaryShopId` method.

Reply:
That's still not correct. This could from the primary shop or from the individual shops depending on marketplace settings.